src/app/ipv/middleware.js

```diff
@@ -187,7 +192,31 @@ module.exports = {
   },
   handleJourneyPage: async (req, res, next) => {
     try {
+      const currentEnvironment = process.env.NODE_ENV;
       const { pageId } = req.params;
+
+      if (
+        currentEnvironment === "development" &&
+        req.session.visitedAllTemplates
+      ) {
+        if (pageId === "page-ipv-reuse") {
+          let userDetailsResponse = samplePersistedUserDetails;
```

Comment:
Does this need to be `let`? (I may have missed the reassignment in all fairness)